From 53cc912691a0f385c547a50868d864ccb73beb0a Mon Sep 17 00:00:00 2001
From: Arne Fitzenreiter <arne_f@ipfire.org>
Date: Tue, 9 Nov 2021 08:52:11 +0100
Subject: [PATCH 1/8] rockchip: dt: nanopi-r2s: change button from restart to
 power

this is to proper shutdown via acpi with this button.

Signed-off-by: Arne Fitzenreiter <arne_f@ipfire.org>
---
 arch/arm64/boot/dts/rockchip/rk3328-nanopi-r2s.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3328-nanopi-r2s.dts b/arch/arm64/boot/dts/rockchip/rk3328-nanopi-r2s.dts
index a4399da7d8b1..2c656e193581 100644
--- a/arch/arm64/boot/dts/rockchip/rk3328-nanopi-r2s.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3328-nanopi-r2s.dts
@@ -36,9 +36,9 @@ keys {
 		pinctrl-names = "default";
 
 		key-reset {
-			label = "reset";
+			label = "power";
 			gpios = <&gpio0 RK_PA0 GPIO_ACTIVE_LOW>;
-			linux,code = <KEY_RESTART>;
+			linux,code = <KEY_POWER>;
 			debounce-interval = <50>;
 		};
 	};
-- 
2.39.5

